src/lib/pipeline.ml

```diff
@@ -164,13 +168,20 @@ module Full (Bfx: Extended_edsl.Semantics) = struct
   open Option
   module Stdlib = Biokepi.EDSL.Library.Make(Bfx)
 
-  let to_bam ~parameters ~reference_build input =
-    let list_of_inputs = Stdlib.bwa_mem_opt_inputs input in
-    List.map list_of_inputs ~f:(Bfx.bwa_mem_opt ~reference_build ?configuration:None)
+
+  let to_bam ~parameters ~reference_build samples =
```

Comment:
Yeah, this is only DNA BAMs (will fix this name). We do run the indel realigner (in the `final_bams` function, also poorly named).